app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -253,21 +253,21 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
 
 
     $scope.displayUpdating = ->
-      StatusMessage.display 'progress', 'Saving...'
+      StatusMessage.display 'progress', t("saving")
 
 
     $scope.displaySuccess = ->
-      StatusMessage.display 'success', 'Changes saved.'
+      StatusMessage.display 'success',t("products_changes_saved")
 
 
     $scope.displayFailure = (failMessage) ->
-      StatusMessage.display 'failure', "Saving failed. #{failMessage}"
+      StatusMessage.display  'failure', t("products_update_error_msg") + "#{failMessage}"
 
 
     $scope.displayDirtyProducts = ->
       if DirtyProducts.count() > 0
-        message = if DirtyProducts.count() == 1 then "one product" else DirtyProducts.count() + " products"
-        StatusMessage.display 'notice', "Changes to #{message} remain unsaved."
+        message = if DirtyProducts.count() == 1 then t("one_product") else DirtyProducts.count() + t("products")
```

Comment:
Here is a risk of a lost space.
It would probably be a good idea to store `DirtyProducts.count()` in a variable as well.